### DISCUSSION TITLE
HLS stream Error

### DISCUSSION THREAD

**Discussion by Arkgelskad** [f1multiviewer/issue-tracker#15] [Q&A] [answered]
When looking through the 2010 and 2008 seasons, and replaying races,
![Screenshot 2022-06-10 120825](https://user-images.githubusercontent.com/107276241/173133873-084205e1-4346-4ff4-8696-08770eeb3b1d.png)
 I get the message: 

(SOURCE_HLS_STREAM_ERROR)

**Comment by f1multiviewer** [comment]
I did some digging, but I'm not quite sure what's going on there. I can reproduce this though, and I can see it works fine on the website. I'll check back on this later if I can.

**Reply by f1multiviewer** [reply]
Actually I gave it another go and I know what's going on. F1MV tries to get the highest quality possible, but has a minimum quality that it needs to meet, otherwise it'll never switch to that quality (to avoid very low quality streams). Turns out, 2010 is in pretty poor quality and none of the variants/renditions meet the strict standards. I'll see if I can somehow add custom adaptation so it still works. I need to fix it anyway because people want to change to lower qualities (which is also not possible now)

**Comment by f1multiviewer** [comment, marked as answer]
This has been fixed!
